src/main/java/at/hannibal2/skyhanni/test/SkyHanniConfigSearchResetCommand.kt

```diff
@@ -107,13 +108,42 @@ object SkyHanniConfigSearchResetCommand {
         val shimmy = Shimmy.makeShimmy(root, list) ?: return "§cCould not change config element '$term', not found!"
         return try {
             shimmy.setJson(element)
-            "§eChanged config element $term."
+            "§eChanged config element $term to $rawJson."
         } catch (e: Exception) {
             ErrorManager.logError(e, "Could not change config element '$term' to '$rawJson'")
             "§cCould not change config element '$term' to '$rawJson'"
         }
     }
 
+    private suspend fun toggleCommand(args: Array<String>): String {
+        if (args.size < 2 || args.size == 3) return "§c/shconfig toggle <config name> [value 1] [value 2]"
+
+        val path = args[1]
+        val rawJson1 = if (args.size > 2)
+            if (args[2] == "clipboard") OSUtils.readFromClipboard() ?: return "§cClipboard has no string!" else args[2]
+        else "true"
+        val rawJson2 = if (args.size > 2)
+            if (args[3] == "clipboard") OSUtils.readFromClipboard() ?: return "§cClipboard has no string!" else args[3]
+        else "false"
```

Comment:
no clipboard for this

Reply:
Why?

Reply:
Why is it useful to have clipboard here

Reply:
Consistency?
Why is it bad to have clipboard here?

Reply:
I need to look at the full code later, but it looks like workarounds with clipboard logic. Im a fan of allowing to have `/shconfig set path clipboard` to work with json directly, same as outputting via `/shconfig load path`. Just Parse the clipboard content as string to a json element and try to apply in the config element. Same for reading